lib/active_snapshot/config.rb

```diff
@@ -0,0 +1,20 @@
+class ActiveSnapshot::Config
+  class << self
+    attr_accessor :storage_method
+  end
+
+  def self.setup(&block)
+    @storage_method = :yaml
```

Comment:
Id prefer if we worked with string instead of symbol for this class/elsewhere

Reply:
May I ask why? I'm rather new to Ruby. Thought that symbols are preferred in such usage because of immutability and faster comparison

Reply:
I think the performance difference is a moot point essentially. String comparisons are fast too.

There are a few reasons one may like strings more

- Strings are common to all programming languages and sometimes its nice to just keep in the programmers mindset rather than the rubyism mindset
- I personally am not a fan of using symbols outside of parenthesis such as `val = :yaml`

Im sure you could post this question on reddit /r/ruby and get some fairly interesting replies.